### PR TITLE
Allow devs access to data in dev/preprod

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Ticket Number

AS-

## Description of change

Allow devs access to MongoDB and Blob Storage in dev/preprod clusters

## Checklist

- [x] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Comment by MrSimonEmms** [comment]
Tested with @Fenisay's help. Can someone check my updated docs read correctly as well please?

**Review by Fenisay — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -421,3 +423,27 @@ Typically, you will need to add some identifying detail to associate the logs to
 const log = logger.child({ someId: uuid.v4() });
 log.info('info_message');
 ```
+
+## Data Access
+
+In Dev and PreProd, the data is available for developers to interrogate as personally-identifiable information should not
+be used in these environments. To access the data, please use the [Azure Portal](https://portal.azure.com).
```
[the remark this answers is not in the document]
Comment (MrSimonEmms) [reply README.md:430]: Yeah, it's hardly Shakespeare. It's not even Robert Greene. How about:

> Developers are granted access to the data stores in the Dev and PreProd environments. No sensitive data is stored in these environments making is appropriate for developer access.
>
> To access the data, please use the Azure Portal

[the remark this answers is not in the document]
Comment (a6software) [reply README.md:430]: swedish_chef

[the remark this answers is not in the document]
Comment (MrSimonEmms) [reply README.md]: Teamwork! https://www.youtube.com/watch?v=pGFGD5pj03M&ab_channel=Channel4

File: README.md [hunk 2]
```diff
@@ -421,3 +423,29 @@ Typically, you will need to add some identifying detail to associate the logs to
 const log = logger.child({ someId: uuid.v4() });
 log.info('info_message');
 ```
+
+## Data Access
+
+Developers are granted access to the data stores in the Dev and PreProd environments. No sensitive data is stored in 
+these environments making is appropriate for developer access.
```
Comment (a6software) [inline README.md:430]: > making is

slight typo

**Review by a6software — APPROVED** [review]

**Pull request merged**